frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -397,13 +403,20 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
               {!validAllowedBusses.has(bus) && (
                 <FormSelectOption
+                  isDisabled
```

Comment:
is this really working for you? Currently it lets me select even invalid values

Reply:
it does, look at the screenshot I added to one of the first comments

Reply:
you are right. It works! I had a wrong branch - sorry about that